plasmapy/atomic/classes.py

```diff
@@ -40,6 +39,12 @@
 class Particle():
     r"""A class for individual particles or antiparticles."""
 
+    def __repr__(self):
+        return f'<Particle "{self._particle_symbol}">'
+
+    def __str__(self):
+        return f"{self._particle_symbol}"
```

Comment:
Good idea on the dunder methods. I tend to like my class to have `__init__` at the top, as that's what gets executed first and so on, but I guess this is fine as well.

Makes me wonder... we could also overload addition or other operators for these particles. Something something reactions?

Reply:
@Cadair had brought this idea up a while ago too, to have something like:
```Python
>>> Be-8 > 2 * He-4
<whatever the energy output is>
```
I think he said that the thing to be careful about was the order of operations, in that the `-4` would signify assigning a mass number to an element.  This might be worth creating an issue for, since it won't happen before the v0.1.0 feature freeze.  This would be neat functionality, though.

Reply:
As an aside, Be-8 really doesn't like being Be-8.  It's half-life is 6.7e-17 seconds before it turns into two alpha particles.  Despite that incredibly short half-life, Be-8 is an important part of the triple-alpha process for nucleosynthesis in stars.